src/main/java/core/basesyntax/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculateArea {
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/210249357-878f9692-a372-42ec-8050-8070e2cdd3f2.png)

```suggestion
public interface AreaCalculator {
```